src/components/CookieBanner.tsx

```diff
@@ -73,6 +73,12 @@ export const CookieBanner = ({ segmentDestinations }: { segmentDestinations: str
   const [show, setShow] = useState(false);
 
   useEffect(() => {
+    // launch google optimize
+    if (window) {
+      window.dataLayer = window.dataLayer || [];
```

Comment:
Do you think this is more readable?
```suggestion
if (!window.dataLayer) {
        window.dataLayer = [];
      }
```
Not a strong opinion, feel free to change if you think it improves readability